### PR TITLE
Stunner update for PR process

### PR DESCRIPTION
**Pull request by hasys**
Hi guys,

there are still some questions during Monday's meetings about how to proceed with PRs. So it is a gentle reminder that we have described step by step process how to simplify review for all parties (authors, Developers reviewers, QE reviewers, UX/DOCs reviewers if needed).

 I also extended this step by step guide with new Github feature Draft PRs.

Please, approve this PR if you agree with steps and willing to use it in your day to day job and propose your changes if you have any ideas.

Thank you!

### CODE REVIEW

File: kie-wb-common-stunner/README.md [hunk 1]
```diff
@@ -31,14 +31,16 @@ PR process
...
   - Wait for the results of project Jenkins and full downstream build Jenkins results.
   - If Jenkins results are not green, check the results and if the issues are related to your changes, fix them and re-trigger builds.
     - Repository build will be triggered automatically when you push your changes, but Full downstream build should be re-triggered manually.
   - If Jenkins builds are not green but you are sure it is not related to your changes, find the author of the failing tests and ping him/her in the PR comments.
   - Ensure that your PR passed Sonar Cloud gates as well. If you do not agree with the issues reported by Sonar or your tests are running but not measured correctly, put a comment to your PR.
   - If Jenkins builds are green or you are sure that failed tests are not related to your changes and Sonar Cloud gates are passed, download Business Central from Full downstream build and test it locally against Acceptance Criteria / Steps to reproduce. Sometimes the issue is fixed in the showcase but still reproducible in the product.
```
Comment (mareknovotny) [inline kie-wb-common-stunner/README.md]: I would add that if tests are not related to the change we should try to file a jira and/or ignore the test in the Test class source code

File: kie-wb-common-stunner/README.md [hunk 2]
```diff
@@ -31,14 +31,16 @@ PR process
 - During commit add appropriate jira issue ID (JBPM, RHPAM or RHDM only) at the beginning of the commit message.
 - Before pushing to Github execute `mvn clean package -Dfull` to execute checkstyle and spotbug plugins locally.
 - When creating multi-repository PR ensure that all repositories have the same branch name otherwise our Jenkins will build PRs separately.
+- Create [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) instead of regular.
+- If you need to check some particular points (on the code or runtime level) to finish the task or ensure that everything is fine, ask a particular person in the commit messages before you will perform final runtime tests.
```
Comment (LuboTerifaj) [inline kie-wb-common-stunner/README.md]: Why you should ask people in commit messages?

Comment (hasys) [reply kie-wb-common-stunner/README.md]: Yea, it is about PR messages, not commit

File: kie-wb-common-stunner/README.md (hunk 1 above)
Comment (hasys) [reply kie-wb-common-stunner/README.md]: I think it should be done by the author/maintainer of the test (in the process we should ping them). Because they should know if this issue already reported or it is new one or somebody already working on it. 

But in general, good point. Probably we can consider this improvement in the future:
- if test is not related to the change - create PR with `@Ignore` and create a Jira issue. As PR reviewer should be author/maintainer of the test.

**Comment by romartin** [comment]
@hasys lemme know when it's ready to merge!

**Comment by hasys** [comment]
Hi @LuboTerifaj, @treblereel, @lazarotti,

are you agree with steps in general? Do you have any notes/suggestions you would like to discuss?

Thank you!

**Comment by lazarotti** [comment]
I know it was already merged, but good to me anyway :) :+1:
